CHANGES.rst

```diff
@@ -187,6 +187,10 @@ astropy.io.misc
 astropy.io.fits
 ^^^^^^^^^^^^^^^
 
+  Create magic methods ``__copy__()`` and ``__deepcopy__`` on an ``HDUList``
```

Comment:
Missing a list bullet, and I think that parenthesis are unnecessary for method names (or at least it should consistent).

Reply:
Ah, I see. Gotta fix that.